.travis.yml

```diff
@@ -3,18 +3,29 @@ language: rust
 cache: cargo
 
 before_install:
-    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-3.0.14.tgz
-    - tar xvf mongodb-linux-x86_64-3.0.14.tgz
-    - mv mongodb-linux-x86_64-3.0.14 3.0.14
-    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-3.2.11.tgz
-    - tar xvf mongodb-linux-x86_64-3.2.11.tgz
-    - mv mongodb-linux-x86_64-3.2.11 3.2.11
+    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1204-3.0.14.tgz
+    - tar xvf mongodb-linux-x86_64-ubuntu1204-3.0.14.tgz
+    - mv mongodb-linux-x86_64-ubuntu1204-3.0.14 3.0.14
+    - wget http://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1204-3.2.11.tgz
+    - tar xvf mongodb-linux-x86_64-ubuntu1204-3.2.11.tgz
+    - mv mongodb-linux-x86_64-ubuntu1204-3.2.11 3.2.11
 
 script:
-    - mkdir -p ./data/db ./data/db2
-    - 3.0.14/bin/mongod --fork --nopreallocj --dbpath ./data/db --syslog --port 27017
```

Comment:
I'm not completely sure why we were doing `--nopreallocj` beforehand; I think back when we first started, I was doing that locally to cut down on the startup time, but this isn't really necessary for Travis, as the 3.0 `mongod`'s are starting up in the background while the code is compiling and 3.2+ uses WiredTiger by default (which doesn't preallocate).